arwen/hosttest/execution_test.go

```diff
@@ -2429,16 +2478,16 @@ func TestExecution_AsyncCall_ChildFails(t *testing.T) {
 			WithFunction(parentPerformAsyncCall).
 			WithGasProvided(1000000).
 			WithArguments([]byte{1}).
-			WithCurrentTxHash([]byte("txhash")).
+			WithCurrentTxHash(txHash).
 			Build()).
 		WithSetup(func(host arwen.VMHost, stubBlockchainHook *contextmock.BlockchainHookStub) {
 			host.Metering().GasSchedule().ElrondAPICost.AsyncCallbackGasLock = 3000
 		}).
 		AndAssertResults(func(host arwen.VMHost, stubBlockchainHook *contextmock.BlockchainHookStub, verify *test.VMOutputVerifier) {
 			verify.Ok().
-				GasUsed(test.ParentAddress, 997169).
+				GasUsed(test.ParentAddress, 997052).
 				GasUsed(test.ChildAddress, 0).
-				GasRemaining(2831).
+				GasRemaining(2948).
```

Comment:
The diff shows that the previous `GasRemaining` was `2831`. What was the explanation for this increase?